### PR TITLE
Add 'Copy URL' action to assets

### PR DESCRIPTION
**Pull request by duncanmcclean**
This pull request implements statamic/ideas#266 by adding a 'Copy URL' to the Assets Listing Table. 

Thanks to Jason's changes on my previous PR, #5854, this is now possible!

### CODE REVIEW

File: src/Actions/CopyAssetUrl.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    public function visibleToBulk($items)
+    {
+        return false;
+    }
+
+    public function authorize($authed, $user)
+    {
+        return $authed->can('sendPasswordReset', $user);
```
Comment (edalzell) [inline src/Actions/CopyAssetUrl.php:28]: what does this have to do with the password reset?

Comment (duncanmcclean) [reply src/Actions/CopyAssetUrl.php:28]: Hah, good point. Copied from my other PR, will update it tomorrow.

**Comment by duncanmcclean** [comment]
Fixed the permisison check! It'll now check if the user has access to `view` the current asset.

**Comment by jackmcdade** [comment]
Hmm, the URL it copies is always the URL of the page I'm on (e.g. http://sandbox.test/cp/assets/browse/assets instead of http://sandbox.test/assets/img/logo.png). 🤔

**Comment by duncanmcclean** [comment]
I actually never got the copy/paste stuff to work locally, tried a couple of things. I copied what Jason did in my previous PR, #5854 to make it work....

**Comment by jackmcdade** [comment]
Ah so it might be a core bug, gotcha.

**Comment by jasonvarga** [comment]
I've fixed that in #5912
